File: src/main/java/com/Jay/Hello.java
package com.Jay;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
        Persion p =new Persion();
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
//        int age = 19;
//        char c = 'A';
//        byte b = 120;
//        float weight = 45.5f;
//        float height = 1.7f;
//        boolean adult = true;
//        boolean enroll = false;
//        String name = "Tom";
    }
}

File: src/main/kotlin/com/Jay/Hello.kt
package com.Jay

fun main() {
//    println("Hello kotlin")
//    Human().hello()
    val h = Human(66.5f,1.7f)
    h.hello()
    println(h.bmi())
//    var age = 419
//    age = 20
//    var wei = 66.5f
//    var name : String
//    name = "JAU"

}

class Human(var weight: Float, var height: Float){
    fun bmi(): Float{
        val bmi = weight/(height * height)
        return bmi
    }
    fun hello(){
        println("Hello kotlin")

    }
}